inspec/a2-api-integration/controls/ingest-job-scheduler.rb

```diff
@@ -93,6 +93,7 @@ def refresh_elasticsearch()
           ).http_status
         ).to eq 200
 
+        sleep 10
```

Comment:
Did you find that you needed this sleep? The `refresh_elasticsearch` should ensure all the data is updated.

Reply:
I'll look into this, but IIRC, the sleep is now needed because the job worker might take a second to pick up the task when you trigger it.